### DISCUSSION TITLE
APIMAN 3 DOCKER-COMPOSE ON AWS EC2

### DISCUSSION THREAD

**Discussion by DevOps7085** [apiman/apiman#2434] [Help]
Hi,

I have used docker-compose instruction for installing apiman3 on AWS ec2. When i start docker-compose it will show me successful in docker ps command mean all container are up and running. But when I accessing on ec2 public ip it will not accessible. I had change .env file in that I had changed APIMAN_HOSTNAME: myec2-publicip, but didn't get any luck.

Do you have any suggestion for me. This will great help.
Thanks

**Comment by msavy** [comment]
I assume you've tried deploying locally and everything works OK?

I wonder if you need to customise the Traefik proxy configuration for EC2? You might want to look into the logs on the Traefik container and read a bit about Traefik and EC2.

Do you see anything unusual in the logs on any of the containers?

**Reply by msavy** [reply]
Also, please check out this tutorial https://aws.amazon.com/blogs/containers/deploy-applications-on-amazon-ecs-using-docker-compose/

Please let the community know how you get on.
